tailwind.config.ts

```diff
@@ -113,8 +113,7 @@ const config = {
       },
     },
   },
-  // eslint-disable-next-line unicorn/prefer-module
   plugins: [require("tailwindcss-animate")],
 } satisfies Config;
 
-export default config;
+export default config;
```

Comment:
why are you adding a new line here as a change?

Reply:
It's because I the dev has been updated